app/workers/commit_monitor_handlers/commit_range/rubocop_checker/message_builder.rb

```diff
@@ -124,4 +124,9 @@ def rubocop_version
   def hamllint_version
     HamlLint::VERSION
   end
+
+  def yamllint_version
+    _out, _err, status, _thread = Open3.popen3("yamllint -v")
```

Comment:
I think the variables are named incorrectly, and also it doesn't closed the in and out pipes, so I don't think this is what you want to do...

According to the [docs](https://ruby-doc.org/stdlib-2.4.2/libdoc/open3/rdoc/Open3.html#method-c-popen3), popen3 returns `stdin, stdout, stderr, wait_thr`.  So I gather you are just trying to read the version from STDERR.  If so, it's probably preferable to use capture3:

```ruby
_out, err, _ps = Open3.capture3("yamllint -v")
err.chump.split.last
```

Reply:
This is what happens when I scan three different articles on how to do this 😆